src/abstractsparsearrayinterface.jl

```diff
@@ -102,6 +106,10 @@ end
   SparseArraysBase.storedvalues(::T)
 end
 
+@derive (T=SubArray,) begin
+  SparseArraysBase.isstored(::T, ::Int...)
+end
```

Comment:
Is this even necessary? I thought `isstored` is already an interface function by default, see [here](https://github.com/ITensor/SparseArraysBase.jl/blob/d741b7fd473b7ca568bc94c2e45042cb6683324c/src/abstractsparsearrayinterface.jl#L97)

Reply:
You're right, I got confused and read to quickly and thought those definitions where just for `Array` (which obviously wouldn't make sense).